frontend/.env

```diff
@@ -0,0 +1,4 @@
+#do not add sensitive information before adding to .gitignore
+
+#full path imports
+NODE_PATH=src/
```

Comment:
@briaguya you ok with the .env file approach? We can also include this NODE_PATH definition in package.json in all the appropriate build commands. 
IF we go the .env approach, how do you feel about defining REACT_APP_AWS_ENDPOINT='http://localhost:5000/graphql there.

Reply:
I'm cool with the .env file approach, I'll make sure everything about it works well with travis.